CMakeLists.txt

```diff
@@ -78,24 +78,25 @@ if (BUILD_TEASER_FPFH)
     endif()
 endif ()
 
+include(FetchContent)
+
 # googletest
-configure_file(cmake/GoogleTest.CMakeLists.txt.in googletest-download/CMakeLists.txt)
```

Comment:
@ThibeauSR I tried wrapping this line with `if-endif` but it didn't have any effect. `configure_file` always occurs so it could conflict with our gtest version. `FetchContent` is a better alternative to it.